agimus_controller/utils/plots.py

```diff
@@ -138,18 +133,21 @@ def plot_control(self):
 
     def display_path(self):
         """Display in Gepetto Viewer the trajectory found with crocoddyl."""
+        if self.rmodel.existJointName("panda_joint7") or self.rmodel.existJointName(
+            "panda2_joint7"
+        ):
+            environnement_pose = [0, 0]  # value of gripper joints
+        elif self.rmodel.existJointName("wrist_3_joint"):
+            environnement_pose = [-0.2, 0, 0.02, 0, 0, 0, 1]  # pose of the ball
         for x in self.croco_xs:
-            if self.ball_init_pose is not None:
-                self.v(list(x)[: self.nq] + self.ball_init_pose)
-            else:
-                self.v(list(x)[: self.nq] + [0, 0])
+            self.viewer(list(x)[: self.nq] + environnement_pose)
             time.sleep(self.DT)
 
     def print_final_placement(self):
         """Print final gripper position for both hpp and crocoddyl trajectories."""
         q_final_hpp = self.whole_x_plan[-1][: self.nq]
         hpp_placement = get_ee_pose_from_configuration(
-            self._rmodel, self._rdata, self._last_joint_frame_id, q_final_hpp
+            self.rmodel, self._rdata, self._last_joint_frame_id, q_final_hpp
```

Comment:
why removing the underscore before the variable?

Reply:
For me the class `MPCPlots` is a class for debuging so sometimes I would like to see what robot model it's using from outside the class

Reply:
Ok, then maybe a simple getter could be appropriate?